e2e-tests/cjs/governance.test.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @jest-environment jsdom
+ */
+
+const nns = require("../../dist"); // in real use cases:  require("@dfinity/nns");
+const { AccountIdentifier, GovernanceCanister } = nns;
+
+test("Should be able to list known neurons", () => {
+  let gov = GovernanceCanister.create();
```

Comment:
Is "mainnet" production?

Isn't it dangerous to run tests against production?

Reply:
The other option would be to deploy a testnet and run the tests against it, right? It doesn't sound trivial.

Isn't the end2end test that @nmattia has done helpful for this?

Reply:
These tests are runnable against any network, be it local, testnet or production.  I explicitly want to be able to run tests against any network.  For CI, yes, an ephemeral network is good.

Running against production is fine, but might cost the caller money if they supply a wallet and they buy neurons.  No wallet, no risk but financial transaction tests won't pass.  Readonly queries will be fine.

Note: I have not yet updated the governance tests to take an actor; in the Ledger tests an actor is provided and that points to whichever network has been assigned.